src/gui/advancedsettings.cpp

```diff
@@ -97,6 +97,8 @@ enum AdvSettingsRows
 AdvancedSettings::AdvancedSettings(QWidget *parent)
     : QTableWidget(parent)
 {
+    // frame style is set by outer QStackedWidget
+    setFrameStyle(QFrame::NoFrame);
```

Comment:
Sure!
[before](https://user-images.githubusercontent.com/9395168/29277313-dd137230-8143-11e7-9292-03141e3f390f.png)
[after](https://user-images.githubusercontent.com/9395168/29277315-dd5c2d0e-8143-11e7-98f0-bea4ac47a0d2.png)
[for comparison](https://user-images.githubusercontent.com/9395168/29277314-dd1b1d3c-8143-11e7-9f02-e9f540249cd8.png)